### ISSUE TITLE
[Enhancement] VoiceOver doesn't register an error in a FormField

### ISSUE THREAD

**Issue by MortenGregersen** [kirbydesign/designsystem#944]
*NOT Prioritized*
- [ ] For optimal support for VoiceOver this is needed.

*NOT UX Refined*
- [x] Doesn't need any UX.

*NOT Tech Refined*
- [x] Decide how the application can tell the `FormFieldComponent` when a message is an error.

**Is your enhancement request related to a problem? Please describe.**
When using the `FormFieldComponent` and setting the `message` property with an error (eg. then submitting the form), VoiceOver isn't notified about the error and the user needs to navigate back to the field to read the error message.

**Describe the solution you'd like**
When the `message` is set with an error message the `alert` role should be set on the message element: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role

**Comment by MortenGregersen** [comment]
A `errorMessage` property to `FormFieldComponent` and render it with a `role="alert"`
